apps/frontend/src/locales/en/translation.json

```
@@ -219,42 +219,52 @@
     "deleteSite": "Delete site"
   },
-  "settings": {
-    "title": "Settings",
-    "subtitle": "Configure your paragliding dashboard",
-    "tabs": {
+  "settings": {
+    "title": "Settings",
+    "subtitle": "Configure your paragliding dashboard",
+    "description": "Preferences are applied immediately to the relevant pages and kept in this browser.",
+    "autoSave": "Auto-save",
+    "saved": "Applied",
```

Biome (2.4.15)

[error] The key saved was already declared.

(lint/suspicious/noDuplicateObjectKeys)